docs/whats-new/zowe-v3-frequently-asked-questions.md

```diff
@@ -0,0 +1,123 @@
+# Zowe V3 Frequently Asked Questions
+
+## General questions
+
+1. When will Zowe V3 be released?
+  
+   Zowe V3 is scheduled to be released on Sept. 28, 2024. Any changes to the plan will be communicated via all available channels.
+
+2. When will Zowe V3 be available for preview?
+  
+   Currently, Zowe V3 pre-release versions are available for the PAX, Zowe CLI, Zowe Client Node.js SDKs, and Zowe Explorer for Visual Studio Code. **[Intellij? Zowe Desktop?]**
+
+## Extender questions
+
+### API Mediation Layer
+
+1. Do we need to move away from passtickets as the method of authentication to the API Mediation Layer?
+    
+   No, we will continue to support passtickets. However, we no longer recommend passtickets because configuration can be inefficient. For every service that uses them, a user must configure a passticket.
+
+2. With the introduction of the new cloud gateway, how is the configuration going to change?
+
+   The cloud gateway configuration for V3 will move to the currently used gateway configuration space (for example, zowe.components.gateway).The configuration that was Zuul specific will be removed and replaced with the Spring Cloud Gateway configuration.
+
+3. How does client certificate authentication work in Zowe V3?
+
+   Northbound authentication accepts client certificates. The API Mediation Layer then transforms the client certificate to another method of authentication (such as a JWT token, SAF IDT, or passticket) and this new method is then accepted by southbound services. Currently, we do not plan to support authentication with client certificates to z/OSMF.
+
+4. Will you identify deprecated functionality such as passtickets? **[where will this functionality be identified? are passtickets really deprecated? first FAQ says no.]**
```

Comment:
@balhar-jakub - please respond

Reply:
Passtickets will be deprecated with V3, but they will be supported and possibly even through to V4